mkosi/__init__.py

```diff
@@ -2170,11 +2177,25 @@ def make_initrd_addon(context: Context, stub: Path, output: Path) -> None:
         "--ro-bind", context.workspace / "initrd", workdir(context.workspace / "initrd")
     ]  # fmt: skip
 
+    if microcodes:
+        # new .ucode section support?
+        check_ukify(
+            context.config,
+            version="256~devel",
+            reason="build addon with .ucode section support",
+            hint=("Use ToolsTree=default to download most required tools including ukify automatically"),
+        )
+
+        for microcode in microcodes:
+            arguments += ["--microcode", workdir(microcode)]
+            options += ["--ro-bind", microcode, workdir(microcode)]
+
```

Comment:
As mentioned in chat, I'd drop this until a proper use case appears, a vendor UKI without microcode is going to be rather broken.